### PR TITLE
#6020 lost precision (python)

### PR DESCRIPTION
**Pull request by rbidas**

### CODE REVIEW

File: beakerx/beakerx/tabledisplay/tabledisplay.py [hunk 1]
```diff
@@ -91,7 +93,10 @@ def convertType(self, object_type, value):
         if object_type == "float64":
             return "double"
         if object_type == "int64":
-            return "integer"
+            if value > (pow(2, 31) - 1) or value < -(pow(2, 31)):
```
Comment (scottdraves) [inline beakerx/beakerx/tabledisplay/tabledisplay.py]: use sys.maxint etc, do not compute these values.

Comment (scottdraves) [reply beakerx/beakerx/tabledisplay/tabledisplay.py]: unclear to me that it's the right value at all.  js supports 56 bits before we have to switch.

Comment (scottdraves) [inline beakerx/beakerx/tabledisplay/tabledisplay.py]: use a standard constant for this eg https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.iinfo.html
if pandas has one that's better.

Comment (rbidas) [reply beakerx/beakerx/tabledisplay/tabledisplay.py]: done. numpy used.

**Pull request merged**